backend/src/models/status.js

```diff
@@ -0,0 +1,44 @@
+const { Model } = require('objection')
+const tableStatuses = 'statuses'
+
+class Status extends Model {
+  static get tableName () {
+    return tableStatuses
+  }
+
+  static get idColumn () {
+    return ['StatusID']
+  }
+
+  static get relationMappings () {
+    const Student = require('./student')
+
+    return {
+      statusID: {
```

Comment:
```suggestion
      students: {
```
We are retrieving a list of `Student`s with a given `StatusID`.